### ISSUE TITLE
Walcha Council: No data received since 21 Sep 2021

### ISSUE THREAD

**Issue by mlandauer** [planningalerts-scrapers/issues#462]
This issue has been **automatically** created by PlanningAlerts for [Walcha Council](https://www.planningalerts.org.au/authorities/walcha)

It uses the scraper [planningalerts-scrapers/multiple_horizon](https://morph.io/planningalerts-scrapers/multiple_horizon)

Only close this issue once the authority is working again on PlanningAlerts. Otherwise a new issue will just automatically created. Also, if there is a duplicate issue close the other one in favour of this one.

**Comment by tseyiokorodudu** [comment]
The Walcha council now uses [the NSW planning portal](https://www.planningportal.nsw.gov.au/map).

**Comment by katska** [comment]
oh cool @tseyiokorodudu good find! It's been around for a while but it didn't look like there was much uptake from councils last time I looked. would you mind seeing if there's an issue to figure out how to get info from the NSW planning portal, and writing one if not? Would be exciting to have a more direct data feed if that's possible in NSW now!

**Comment by tseyiokorodudu** [comment]
@katska according to Matthew, the NSW planning portal is "pretty useless and doesn't provide much usable information for us". I guess that makes it hard to scrape. 

On a second look, I realised the error comes from a URL change and has nothing to do with the NSW portal. I found the new Walcha URL for the development applications. I will just update that now.  

> The Walcha council now uses [the NSW planning portal](https://www.planningportal.nsw.gov.au/map).
